src/components/separators/_horizontal-separator.scss

```diff
@@ -14,7 +14,11 @@ $includeHtml: false !default;
     width: 100%;
 
     &--spaced {
-      margin: (rhythm(1) - $horizontalSeparatorHeight) 0 rhythm(1);
+      margin: rhythm(0.333) 0 rhythm(0.667);
```

Comment:
The sum of margins will be `rhythm(1)`. 
But I believe I need to add `$horizontalSeparatorHeight` to equation. 
Thanks!